subhub/api/payments.py

```diff
@@ -181,6 +191,12 @@ def create_update_data(customer) -> dict:
     :param customer:
     :return: return_data dict
     """
+    for subscription in customer["subscriptions"]["data"]:
+        if subscription["status"] == "incomplete":
+            invoice = stripe.Invoice.retrieve(subscription["latest_invoice"])
+            print(f"invoice {invoice}")
```

Comment:
Print statements shouldn't be included. To avoid having to include/exclude them during local dev, you could use log.debug statements instead.